regress/regress_expected

```diff
@@ -87,7 +87,7 @@ ERROR:  parse error - invalid geometry at character 23
 76|f
 76a|f
 106|BOX3D(0 0 0,7 7 0)
-107|POLYGON((0 0,0 7,7 7,7 0,0 0))
+107|POLYHEDRALSURFACE(((0 0 0,0 7 0,7 7 0,7 0 0,0 0 0)),((0 0 7,0 7 7,7 7 7,7 0 7,0 0 7)),((0 0 0,0 0 7,0 7 7,0 7 0,0 0 0)),((7 0 0,7 0 7,7 7 7,7 7 0,7 0 0)),((0 0 0,0 0 7,7 0 7,7 0 0,0 0 0)),((0 7 0,0 7 7,7 7 7,7 7 0,0 7 0)))
```

Comment:
We should also have tests to catch the potential 3D polygon outputs from this cast.